examples/tts/conf/hifigan/model/validation_ds/val_ds.yaml

```diff
@@ -3,8 +3,8 @@ dataset:
   _target_: "nemo.collections.tts.data.datalayers.AudioDataset"
   manifest_filepath: ${validation_datasets}
   max_duration: null
-  min_duration: null
-  n_segments: -1
+  min_duration: 3
+  n_segments: 66048
```

Comment:
Can you explain why you made these changes?

Reply:
@blisc, validation step faster with this setting (instead of using `n_segments=-1`), also it can potentially produce OOM if dataset has long samples. By the way, `validation_ds/val_ds_finetune.py` has exactly this setting.